### PR TITLE
yamidecode: window should be created if formatInfo is set in decoder-…

### PR DESCRIPTION
**Pull request by zhongcong**
…>start().

Signed-off-by: Zhong Cong <congx.zhong@intel.com>

### CODE REVIEW

File: tests/decode.cpp [hunk 1]
```diff
@@ -103,6 +103,13 @@ int main(int argc, char** argv)
     }
 
     status = decoder->start(&configBuffer);
+
+    formatInfo = decoder->getFormatInfo();
+    if (formatInfo->width && formatInfo->height) {
+        if (!output->setVideoSize(formatInfo->width, formatInfo->height))
```
Comment (xuguangxin) [inline tests/decode.cpp]: could we put setVideoSize to this if?
if (input->getWidth() && input->getHeight()) {
        configBuffer.surfaceWidth = input->getWidth();
        configBuffer.surfaceHeight = input->getHeight();
    }

Comment (zhongcong) [inline tests/decode.cpp]: @xuguangxin the width & height set to video size is aligned. if put setVideoSize to previous if, we should do the align for the width and height.

**Comment by zhongcong** [comment]
bat test h264&vp9 pass.
vp8 regressionTest will let the QA to do it.

**Comment by xuguangxin** [comment]
+1

**Comment by xuguangxin** [comment]
merged as dbcd4e853855d5a38136886c49d3d377b1c692aa

**Pull request closed without merging**